kore/src/Kore/Repl/State.hs

```diff
@@ -669,44 +669,43 @@ createOnePathClaim (claim, cpattern) =
         }
 
 conjOfOnePathClaims
```

Comment:
Was this not also meant to be polymorphic over the claim type, like the other changes?